packages/resource-deployment/templates/rest-api-templates/model-products.template.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "apimServiceName": {
+            "type": "string"
+        }
+    },
+    "resources": [
+        {
+            "properties": {
+                "description": "Subscribers will be able to run 5 calls/minute up to a maximum of 100 calls/week.",
```

Comment:
fis this description

Reply:
This is the default description. Leaving it unchanged since we don't use it.